kubesae/image.py

```diff
@@ -20,12 +20,17 @@ def generate_tag(c):
 
 
 @invoke.task(pre=[generate_tag])
-def build_image(c, tag=None):
+def build_image(c, tag=None, service=None):
     """Build Docker image using docker-compose."""
     if tag is None:
         tag = c.config.tag
-    # build app container
-    c.run("docker-compose build app", echo=True)
+    if service is None:
+        if "service" in c.config:
+            service = c.config.service
+        else:
+            service = "app"
+    # build service container
+    c.run(f"docker-compose build {service}", echo=True)
```

Comment:
I'm wondering why we're using docker-compose to build a docker image instead of straight `docker build .`?  That wouldn't require reference to how a particular project's docker-compose config was set up.